src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalTests.cs

```diff
@@ -814,6 +815,61 @@ public void TestStringLiteralWithBadUnicodeEscape()
             Assert.Equal(value, token.ValueText);
         }
 
+        [Fact]
+        [Trait("Feature", "Literals")]
+        public void TestStringLiteralWithEscape_CSharp12()
+        {
+            var text = """
+                "\e"
+                """;
+            var value = "\u001b";
+            var token = LexToken(text, TestOptions.Regular12);
+
+            Assert.NotEqual(default, token);
+            Assert.Equal(SyntaxKind.StringLiteralToken, token.Kind());
+            Assert.Equal(text, token.Text);
+            var errors = token.Errors();
+            Assert.Equal(1, errors.Length);
+            AssertEx.EqualOrDiff("error CS8652: The feature 'string escape character' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.", errors[0].ToString(EnsureEnglishUICulture.PreferredOrNull));
+            Assert.Equal(value, token.ValueText);
+        }
+
+        [Fact]
+        [Trait("Feature", "Literals")]
+        public void TestStringLiteralWithEscape_Preview()
+        {
+            var text = """
+                "\e"
+                """;
+            var value = "\u001b";
+            var token = LexToken(text, TestOptions.RegularPreview);
```

Comment:
`RegularPreview` is the default. I think you intend to test using `RegularNext`, which will be updated mechanically to `Regular13` once it becomes available.

Reply:
done.  thanks!